Fireworks/Geometry/interface/FWTGeoRecoGeometry.h

```diff
@@ -19,7 +19,7 @@ class FWTGeoRecoGeometry
   struct Info
   {
     std::string name;
-    float points[24]; // x1,y1,z1...x8,y8,z8
+    float points[36]; // x1,y1,z1...x8,y8,z8,...,x12,y12,z12
```

Comment:
@bsunanda and @alja - I'd suggest you try to reduce the need for all corners. IMHO, an event display will not show an HGCal cell on a screen in such detail and the size of the struct could be reduced dramatically.

Reply:
I am trying to make a separate PR with this going back to 8 corners. I am not sure if the wafers rather than cells can be shown in geometry.